packages/tooling/fast-figma-plugin-msft/src/figma/controller.ts

```diff
@@ -11,6 +37,15 @@ export class FigmaController extends Controller {
         }
     }
 
+    public sync(nodeIds: string[]): void {
```

Comment:
Can you add comments describing what "sync" is intended to do?

Reply:
Added at the generic controller level.